intro-to-ruby.rb

```diff
@@ -1,14 +1,24 @@
 #create a string and store it in a variable
-
+y= "chocolate"
+x = "I love #{y} so much."
 #print out the value of the variable you just created
-
+puts x
 #print the sum of 185 and 14
+a= (185)
+b= (14)
 
+puts (a+b)
```

Comment:
could you write this in one line? What's the advantage of storing 185 and 14 as variables?